### PR TITLE
VAN-67270: Pull and fix yamls

### PR DESCRIPTION
**Pull request by OdeclasV**
Resolving numbers:

2 . yaml with missing `phone` step
4 . moving yam with CAPTCHA to /secureimage
5 . blacklisting periods in `phone` step
6 . moving CT Senate Republicans to /recaptcha

### CODE REVIEW

File: members/CusO1f14eac61548261742.yaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
+bioguide: CusO1f14eac61548261742
```
Comment (tomdionesotes) [inline members/CusO1f14eac61548261742.yaml]: another custom target added?

File: members/WAL000038.yaml [hunk 2]
```diff
@@ -1,68 +0,0 @@
-bioguide: WAL000038
```
Comment (tomdionesotes) [inline members/WAL000038.yaml]: did this target leave office or remove their form?

File: members/CusO1f14eac61548261742.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/CusO1f14eac61548261742.yaml]: this is odd, cuz this custom target was added ( and the other target WAL000038 deleted) by Tommy while I was out a couple of weeks ago. I merged these changes from my personal computer while wfh. I'm assuming that merging this PR to master might create a duplicate yaml for CusO1f14eac61548261742 right? If so, should I close this PR and merge again the new changes? (I'm back in the office today)

Comment (tomdionesotes) [reply members/CusO1f14eac61548261742.yaml]: try pulling down the latest contact congress master branch onto your machine, merging the master branch into develop, and then merging the updated develop branch into your `pull-and-fix-yamls` branch.

Comment (OdeclasV) [reply members/CusO1f14eac61548261742.yaml]: Tried that with Anna. We don't understand why this PR is not updated with the latest commits, so going to close this one and open a new PR with the same changes.

**Pull request closed without merging**